thingsvision/thingsvision.py

```diff
@@ -66,24 +66,19 @@ def get_parsers():
         help="Path to directory containing images. (default: ./images)",
         default="./images",
     )
-    parser_extract.add_argument(
-        "--class-names",
-        type=str,
-        help="optional list of class names for class dataset. (default: None)",
-        default=None,
-    )
-    parser_extract.add_argument(
-        "--file-names",
-        type=str,
-        help="optional list of file names for class dataset. (default: None)",
-        default=None,
-    )
```

Comment:
References to these also need to probably be removed from ImageDataset in  utils/data/dataset.py if we're taking them out of action here. Actually, I'm not 100% sure why they are being removed, specifying the ordering of an `ImageDataset` seems potentially useful.

Reply:
I guess the thinking here is that the CLI is the most basic way to get to CNN features from a directory of images (which will create a list of the image order anyway), so if someone wants to specify the order, they should probably just do programatically anyways? In that sense, we also would not need to remove it from the ImageDataset.

Reply:
Having the possibility to add optional class and file names is pretty useful and I would not remove it from the dataset class. OTOH it's quite cumbersome to append all class or file names in your shell to the respective argument for using the CLI. No one whose intention is to perform quick and basic extraction would do that. At least I wouldn't.